tests/encrypted_lvm.spec.ts

```diff
@@ -14,37 +15,69 @@ test.describe('The main page', () => {
     });
 
     test('Installation test with encrypted lvm file system', async ({ page }) => {
+        // optional actions done on the page
+        const actions = Object.freeze({
+          setProduct: Symbol("product"),
+          setMainPage: Symbol("main"),
+          done: Symbol("done")
+        });
+
+        // check for multiple texts in parallel, avoid waiting for timeouts
+        let action = await Promise.any([
+          page.getByText("Product selection").waitFor().then(() => actions.setProduct),
```

Comment:
use the pages and its methods instead of `page.method`.

Reply:
Updated